backstage/demo-roadiehq-values.yaml

```diff
@@ -0,0 +1,160 @@
+# Default values for backstage.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+frontend:
+  enabled: true
+  replicaCount: 1
+  image:
+    repository: roadiehq/backstage-k8s-demo-frontend
+    tag: dev
```

Comment:
This will be changed to latest once I've "promoted" dev to latest (once we're happy to ship this). I'm aware this is a bit pedantic given it's just us using this.